src/Maestro/Microsoft.AspNetCore.ApiVersioning/ApiVersioningOptions.cs

```diff
@@ -16,12 +16,14 @@ public class ApiVersioningOptions
         public Func<TypeInfo, string> GetName { get; set; } = DefaultGetName;
         public IVersioningScheme VersioningScheme { get; set; } = new PathVersioningScheme();
 
+        [CanBeNull]
         public static string DefaultGetVersion(TypeInfo controllerType)
```

Comment:
DefaultVersion can be null?

Reply:
This is a change I pulled in from @alexperovich. Alex?

Reply:
The "version" of a controller can be null. If it is an unversioned controller then it has no version.

Reply:
Answered! I guess, yeah, like the "login" controller and stuff aren't versioned.